vault/overlays/nerc-ocp-infra/backup-job/externalsecret.yaml

```diff
@@ -17,16 +17,16 @@ spec:
   - secretKey: ACCESS_KEY_ID
     remoteRef:
       key: nerc/nerc-ocp-infra/vault/backup/s3
-      property: S3AccessKey
+      property: ACCESS_KEY_ID
```

Comment:
Couldn't most of this all be replaced by a single `dataFrom` element?

```
apiVersion: external-secrets.io/v1beta1
kind: ExternalSecret
metadata:
  name: vault-backup-s3-endpoint
spec:
  secretStoreRef:
    name: nerc-cluster-secrets
    kind: ClusterSecretStore
  refreshInterval: "1h"
  target:
    name: vault-backup-s3-endpoint
    creationPolicy: 'Owner'
    deletionPolicy: "Retain"
    template:
      engineVersion: v2
  dataFrom:
    - extract:
        key: nerc/nerc-ocp-infra/vault/backup/s3
```

That gets us:

```
apiVersion: v1
kind: Secret
metadata:
  name: vault-backup-s3-endpoint
type: Opaque
data:
  ACCESS_KEY_ID: ...
  BUCKET: ...
  ENDPOINT: ...
  SECRET_ACCESS_KEY: ...
immutable: false
```

Reply:
@dystewart It's a great suggetion here. If you do, make sure that the fields in the vault get updated to be the right properties.

Reply:
@larsks good call I actually forgot that you can pull in all the keys at once like this!